report/util.go

```diff
@@ -69,8 +69,14 @@ func formatScanSummary(rs ...models.ScanResult) string {
 				r.FormatServerName(), r.Warnings))
 		}
 	}
-	return fmt.Sprintf("%s\n\n%s", table, strings.Join(
-		warnMsgs, "\n\n"))
+
+	// We don't want warning message to the summary file
+	if config.Conf.Quiet {
+		return fmt.Sprintf("%s\n", table)
+	} else {
```

Comment:
__[Golint]__ if block ends with a return statement, so drop this else and outdent its block ([link](https://sider.review/gh/repos/54808920/pulls/884#issue-4449416))

<sub>posted by Sider</sub>